doc/03-Configuration.md

```diff
@@ -0,0 +1,63 @@
+# Configuration
+
+If Icinga Web has been installed but not yet set up, please visit Icinga Web and follow the web-based setup wizard.
+For Icinga Web setups already running, log in to Icinga Web with a privileged user and follow the steps below to
+configure Icinga Notifications Web:
+
+## Module Activation
```

Comment:
This is not required to show on the website as packages automatically enable modules.

So if you still want to include this, hide it on the website with `if not icingaDocs`.